.github/workflows/release.yml

```
@@ -30,3 +30,13 @@ jobs:
           labels: enhancement
           reviewers: yamafaktory
           branch: performance
+
+  publish-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Here we can also use the latest `v3` (I'll need to bump up that too in the other `yaml` files):
```suggestion
        uses: actions/checkout@v3
```